device/globals/action_menu.gd

```diff
@@ -1,15 +1,11 @@
 var target
```

Comment:
Don't you get complaints about the script not extending a node? Could add `extends Node2D` at the top.

Reply:
Thank you! Yes! I had a commit for that in my master branch originally.

Problem is that this bunch of PRs and issues and remotes and branches are gnawing away at my brain cells bit by bit.

I cherry-picked it into this now.